code/src/main/java/com/google/sps/data/BusinessNumber.java

```diff
@@ -0,0 +1,33 @@
+package com.google.sps.data;
+
+import java.time.Instant;
+
+// Keep track of information needed to contact a business
+public class BusinessNumber implements ContactStatus{
+
+  String nameOfBussiness;
+  long phoneNumber;
+  private long getTimeWhenCalledUnixTimeSeconds;
+  private boolean businessHasBeenCalled = false;
+
+  public BusinessNumber(String nameOfBussiness, long phoneNumber) {
+    this.nameOfBussiness = nameOfBussiness;
+    this.phoneNumber = phoneNumber;
+  }
+
+  // Record time bussiness was contacted
```

Comment:
Feels like you should override setSuccess, getStatus, and getTime... even if they are empty in the interface.

Reply:
This may be my lack of Java experience but I thought we only had to override methods form a superclass?

Reply:
you have to override all methods in an interface too I think

Reply:
Overriding is in general something that you want to do when there are multiple implementations of the same method so that would go for interfaces as well as superclasses

Reply:
okay